studio/model.py

```diff
@@ -64,66 +64,63 @@ def replace_with_env(config):
 def reset_model_providers():
     reset_model()
 
+def get_artifact_store(config):
+    storage_type: str = config['type'].lower()
+
+    if storage_type == 's3':
+        return TartifactStore(S3StorageHandler(config))
+    elif storage_type == 'local':
+        return TartifactStore(LocalStorageHandler(config))
+    else:
+        raise ValueError('Unknown storage type: ' + storage_type)
+
 def get_db_provider(config=None, blocking_auth=True):
 
     db_provider = get_model_db_provider()
-    if not db_provider is None:
+    if db_provider is not None:
         return db_provider
 
-    if not config:
+    if config is None:
         config = get_config()
     verbose = parse_verbosity(config.get('verbose'))
 
+    # Save this verbosity level as global for the whole experiment job:
+    set_model_verbose_level(verbose)
+
     logger = logs.getLogger("get_db_provider")
     logger.setLevel(verbose)
     logger.debug('Choosing db provider with config:')
     logger.debug(config)
 
     if 'storage' in config.keys():
-        artifact_store = get_artifact_store(
-            config['storage'],
-            blocking_auth=blocking_auth,
-            verbose=verbose)
+        artifact_store = get_artifact_store(config['storage'])
     else:
         artifact_store = None
 
     assert 'database' in config.keys()
-    db_provider = None
     db_config = config['database']
     if db_config['type'].lower() == 'firebase':
-        db_provider = FirebaseProvider(
-            db_config,
-            blocking_auth,
-            verbose=verbose,
-            store=artifact_store)
-        artifact_store = db_provider.get_artifact_store()
+        raise NotImplementedError("Firebase is not supported.")
```

Comment:
Is it possible to not drop firebase.  It was being used for the model zoo.  We could resurrect it one day

Reply:
One day, right. OK put back Firebase in the form of FirebaseStorageHandler which will provide storage operations for Firebase endpoints.
Not tested as of now.

Reply:
Thats OK